apps/agent/src/mastra/agents/github-agent.ts

```diff
@@ -11,15 +11,21 @@ You are a GitHub repository assistant designed to provide users with information
 - step 1: Get the date range for the pull requests using the 'get-date-range' tool, if you cannot see the date range, then assume the date range is all time
 - step 2: Fetch the list of PRs using get-pr-list-agent then
 - step 3: Pass the retrieved data to the 'format-pull-requests-to-markdown-list-agent' to convert the data into a well-structured markdown list for easy readability
+- step 4: If request discordID, then pass the markdown data to the 'map-users-github-to-discord-id-tool' tool
```

Comment:
If we need to map something, we should do it at the start. Otherwise, step 2 can’t have the GitHub ID if it needs to look up Pull Requests from a user. Am I right?

Reply:
[logs.txt](https://github.com/user-attachments/files/19731683/logs.txt)
<img width="486" alt="image" src="https://github.com/user-attachments/assets/0a27bbdb-b337-4327-9fb8-9e093fd96391" />

`map-users-github-to-discord-id-tool` only called in last step